app/presenters/entry_presenter.rb

```diff
@@ -0,0 +1,23 @@
+class EntryPresenter
+
+  delegate :title,
+           :summary,
+           :path,
+           to: :entry
+
+  def initialize(entry)
+    @entry = entry
+  end
+
+  def tag(schema)
+    "tag:#{Plek.current.website_root.gsub('http://', '')},#{schema.instance_variable_get(:@feed_options)[:schema_date]}:#{entry.path}"
```

Comment:
`Plek.current.website_root.gsub('http://', '')` isn't going to have the intended effect in environments where we use https - preview, staging and production. I think the right thing to do would be to use a regex: `gsub(%{https?://}, '')`.

Additionally, I think it's worth breaking the `Plek.current.website_root.gsub('http://', '')` and the `schema.instance_variable_get(:@feed_options)[:schema_date]` into private methods so that the line is shorter and it is easier to understand what the tag is made up of.